package.json

```diff
@@ -1,43 +1,43 @@
 {
   "name": "plugin-template-sf-external",
   "description": "A template repository for external sf plugins",
-  "version": "0.0.1",
+  "version": "1.0.0",
   "dependencies": {
-    "@oclif/core": "^2.15.0",
+    "@oclif/core": "^3.11.0",
     "@salesforce/core": "^5.3.20",
-    "@salesforce/sf-plugins-core": "^3.1.20",
-    "tslib": "^2"
+    "@salesforce/sf-plugins-core": "^5.0.1"
   },
   "devDependencies": {
-    "@salesforce/cli-plugins-testkit": "^4.4.7",
-    "@salesforce/dev-scripts": "^6.0.4",
-    "@swc/core": "^1.3.19",
+    "@salesforce/cli-plugins-testkit": "^5.0.4",
+    "@salesforce/dev-scripts": "^7.1.1",
     "@types/inquirer": "^8",
     "eslint-plugin-sf-plugin": "^1.16.15",
-    "oclif": "^3.17.2",
+    "oclif": "^4.0.3",
     "shx": "0.3.4",
     "ts-node": "^10.9.1",
-    "typescript": "^5.2.2"
+    "typescript": "^5.2.2",
+    "@oclif/plugin-command-snapshot": "^5.0.2",
+    "eslint-config-salesforce-typescript": "^3.0.5"
```

Comment:
`eslint-config-salesforce-typescript` is not necessary because dev-scripts is bringing that.